draft-ietf-mls-protocol.md

```diff
@@ -2489,8 +2489,10 @@ A member of the group applies a Remove message by taking the following steps:
 
 * Blank the intermediate nodes along the path from `removed_index` to the root
 
-* Truncate the tree by removing leaves from the right side of the tree until the
-  rightmost leaf node is not blank.
+* Truncate the tree by removing leaves from the right side of the tree until
+  either the rightmost leaf node is not blank or the parent of the rightmost
+  leaf node is not blank.  (If the parent node is not blank, then it is needed
+  to verify parent hashes in the truncated tree.)
```

Comment:
If the parent node is the root node, it can be removed even if it is non-blank, since removing the root node doesn't destroy any valid parent hash chains.

Reply:
Good point.  Updated to reflect this.